views/layout.pug

```diff
@@ -32,4 +32,5 @@ html(lang="en")
         block footer
         script(src='/js/jquery.js')
         script(src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js", integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut", crossorigin="anonymous")
+        script(src='/js/fb-embed.js')
```

Comment:
Shouldn't put this here, this belongs in `index.pug`, because we wouldn't want to pull in facebook's javascript for other pages that don't need it.